actions/npm-check/npm_check.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+
+// trunk-ignore-all(eslint)
+const npmCheck = require("npm-check");
+const YAML = require("yaml");
+
+// TODO(lauri): Find more stable icon path
+const ICON_PATH = "https://avatars.githubusercontent.com/u/6078720";
+
+const pluralize = (count, singular, plural) => {
+  return count == 1 ? singular : plural;
+};
+
+npmCheck({})
+  .then((current) => {
+    const uninstalled = current.get("packages").filter((p) => {
+      return !p.isInstalled;
+    });
+    if (uninstalled.length == 0) {
+      const yaml = YAML.stringify({ notifications_to_delete: ["npm-check"] });
+      console.log(yaml);
+      return;
+    }
+    const uninstalled_count = uninstalled.length;
+    const yaml = YAML.stringify({
+      notifications: [
+        {
+          id: "npm-check",
+          title: "Check npm packages",
+          message: `${uninstalled_count} npm ${pluralize(
```

Comment:
What's our current story on message vs rendered?

Reply:
fallback on message if rendered is not there. However, I am following up to make sure these don't show up on CLI unless explicitly configured to do so.